### PR TITLE
Next step to stable symfony3 update

### PR DESCRIPTION
**Pull request by wbloszyk**
Hello, 

It is not finished PR but i need some feedback or help. 

- Can someone help me update Behat for 2 to 3? Behat 2 is in coflict with Symfony3
- Also help with upgrade phpunit/phpunit to symfony/phpunit-bridge will be awsome
- I will update sonata-project/user-bundle from hotfix to stable version

What do you think about it and this changes?  @phansys @OskarStark 

#614

### CODE REVIEW

**Comment by greg0ire** [comment]
Hey why did you close this? Sorry for not reviewing earlier!

File: src/Sonata/Bundle/DemoBundle/DataFixtures/ORM/LoadNewsData.php [hunk 1]
```diff
@@ -87,7 +90,7 @@ public function load(ObjectManager $manager)
 
 RAW
 ;
-
+*/
```
Comment (greg0ire) [inline src/Sonata/Bundle/DemoBundle/DataFixtures/ORM/LoadNewsData.php]: What's the issue here?

File: .gitignore [hunk 2]
```diff
@@ -2,9 +2,9 @@
 /app/cache/*
 /app/logs/*
 /app/*.cache
-/public/bundles
-/public/uploads
-/public/sitemap*.xml
+/web/bundles
+/web/uploads
+/web/sitemap*.xml
```
Comment (greg0ire) [inline .gitignore]: Why are you reverting this commit?

File: src/Sonata/Bundle/DemoBundle/DataFixtures/ORM/LoadNewsData.php (hunk 1 above)
Comment (wbloszyk) [reply src/Sonata/Bundle/DemoBundle/DataFixtures/ORM/LoadNewsData.php]: <% gist '1552362' 'gistfile1.txt' %> make errors

File: .gitignore (hunk 2 above)
Comment (wbloszyk) [reply .gitignore]: I got /public from web-server. In symfony 3.4 should be /web. Anyway i fix it in new PR. #616

**Pull request closed without merging**